tests/etl/test_perf_data_adapters.py

```diff
@@ -155,6 +155,15 @@ def test_load_generic_data(self):
                                          subtest['value'],
                                          datetime.datetime.fromtimestamp(
                                              self.PUSH_TIMESTAMP))
+        summary_signature = PerformanceSignature.objects.get(
+            suite=perf_datum['suites'][0]['name'], test='')
+        subtest_signature = PerformanceSignature.objects.exclude(
+            suite=perf_datum['suites'][0]['name'], test='')
```

Comment:
I'd probably use `id=summary_signature.id` for the exclude statement. A bit simpler.